### PR TITLE
Check for best practices

### PR DESCRIPTION
**Pull request by Mujeeb4582**
- Please check my project for the best-practices is compiled or not?

### CODE REVIEW

File: src/module/listClass.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+export default class TodoList {
+  allTodos = [];
+
```
Comment (Tamunokuro) [inline src/module/listClass.js:3]: Please use let or const to declare the array

**Review by Tamunokuro — CHANGES_REQUESTED** [review]
# REQUESTED CHANGES ♻️♻️

### Hi @Mujeeb4582  

Great job on the work done so far but there are some issues. You are almost there!
 
almost there

## Highlights
- Professional PR :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/module/listClass.js (hunk 1 above)
Comment (Tamunokuro) [reply src/module/listClass.js:3]: Hello @Mujeeb4582!
Great job so far, just a few edits to move on to the next project


## Highlights
- No linter errors
- Professional git-flow
- Great design

## Required Changes ♻️

Comment (Mujeeb4582) [reply src/module/listClass.js:3]: @Tamunokuro Thanks for your time and feedback. you suggested me that try to declare an array using let or const but I declare the array using `class`, when you are declaring an array using `class` there is no need of `let` or `const`.

**Review by Tamunokuro — APPROVED** [review]

**Review by Tamunokuro — COMMENTED** [review]
# REQUESTED CHANGES ♻️♻️

### Hi @Mujeeb4582  

Great job on the work done so far but there are some issues. You are almost there!
 
almost there

## Highlights
- Professional PR :heavy_check_mark: 
- No linter errors :heavy_check_mark: 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**